example/user-guides/alerting/prometheus-example-rules.yaml

```
@@ -12,3 +12,19 @@ spec:
     rules:
     - alert: ExampleAlert
       expr: vector(1)
```

This is the simplest example that would work:
```suggestion
    rules:
    - alert: ExampleAlert
      expr: vector(1)
      labels:
        severity: critical
    - alert: ExampleAlert
      expr: vector(1)
      labels:
        severity: warning
```

---

Thank you. I will apply your feedback immediately. 👍

---

I have applied your advice and left two lines of annotation in the critical section that are used in AlertmanagerConfig.
What do you think about this change?